client/src/test/java/org/opensearch/ml/client/MachineLearningNodeClientTest.java

```diff
@@ -74,9 +74,7 @@
 import org.opensearch.search.suggest.Suggest;
 
 import java.io.IOException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
let's not import `.*`. You can configure from your intellij : https://doc.nuxeo.com/corg/intellij-idea-setup/